daemon/src/projection.rs

```diff
@@ -667,6 +673,7 @@ impl Tx {
         let cfds_with_quote = cfds
             .into_iter()
             .map(|(_, cfd)| cfd.with_current_quote(quote))
+            .sorted_by(|a, b| Ord::cmp(&a.creation_timestamp, &b.creation_timestamp))
```

Comment:
Did you verify that this sorts descending, i.e. latest timestamp first?

Reply:
its the other way, ill swap it